src/consts/olm_operators.py

```diff
@@ -152,4 +158,15 @@ def get_exception_factory(operator: str):
     if operator == OperatorType.MCE:
         return MCEOperatorFailedError
 
+    if operator == OperatorType.METALLB:
+        return MetalLBOperatorFailedError
+
     return OperatorFailedError
+
+
+def get_operator_properties(operator: str, **kwargs) -> str:
+    if operator == OperatorType.METALLB:
+        if (api_vip := kwargs.get("api_vip")) and (ingress_vip := kwargs.get("ingress_vip")):
+            return f'{{"api_ip": "{api_vip}", "ingress_ip": "{ingress_vip}"}}'
+
+    return ""
```

Comment:
```suggestion
def get_operator_properties(operator: str, **kwargs) -> Optional[str]:
    if operator == OperatorType.METALLB:
        if (api_vip := kwargs.get("api_vip")) and (ingress_vip := kwargs.get("ingress_vip")):
            return f'{{"api_ip": "{api_vip}", "ingress_ip": "{ingress_vip}"}}'

    return None
```
would be more expressive I think